src/Tester/StreamingTests/SampleStreamingTests.cs

```diff
@@ -87,6 +87,15 @@ public void TestCleanup()
             await StreamingTests_Producer_Consumer(streamId, streamProvider);
         }
 
+        [TestMethod, TestCategory( "Nightly" ), TestCategory( "Streaming" )]
+        public async Task SampleStreamingTests_3()
```

Comment:
I've no issue with this test since it uses the extensions in the context of a grain, but it's not really a test targeted at the extension functions.  For instance:
- It probably does not exercise the on complete or on error paths.
- It does not explicitly test the 8 extension functions.
- It does not test the argument validation.
Again, this is not a bad test, it just doesn't provide much code coverage.